hibernate-core/src/main/java/org/hibernate/dialect/identity/H2AlternativeIdentityColumnSupport.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or http://www.gnu.org/licenses/lgpl-2.1.html
+ */
+package org.hibernate.dialect.identity;
+
+/**
+ * Identity column support for H2 2+ versions
+ * @author Jan Schatteman
+ */
+public class H2AlternativeIdentityColumnSupport extends H2IdentityColumnSupport {
```

Comment:
Let's call this `H2FinalTableIdentityColumnSupport`

Reply:
Will do